### DISCUSSION TITLE
Cannot log in

### DISCUSSION THREAD

**Discussion by ContentLTD** [community/community#144548] [Other Feature Feedback, Questions, & Ideas]
Hello, I can't log into GitHub via app verification. Im getting the "CryptoTokenKit -3" error. I can't use another method of verification because it's a security key and the computer has been glitchy with them. Please help. I have reinstalled the app and cleared cahe but still the same issue. ![image](https://github.com/user-attachments/assets/3d662fa0-146a-403f-8830-c13a26f99462)

**Comment by TheGiraffe3** [comment]
Aren't you already logged in?

**Reply by ContentLTD** [reply]
Im logged in on mobile. Im trying to log in on desktop but cant

**Comment by AIBlockchainGuru** [comment]
I'm sorry to hear you're having trouble logging into GitHub on your desktop due to the "CryptoTokenKit -3" error. Since you're already logged in on mobile, here are some steps you can take to resolve the issue on your desktop:
1. Use Mobile for Verification
Since you're logged in on mobile, you can use the GitHub mobile app to verify your login on the desktop. When you attempt to log in on your desktop, GitHub should send a verification request to your mobile app. Follow these steps:
On your desktop, enter your username and password.
When prompted for verification, check your mobile app for a notification or code.
Enter the code displayed in the app into your desktop browser.
2. Check Security Key Functionality
If you're using a security key, ensure that it is functioning properly. Sometimes, issues with the key can cause errors during the login process. If possible, try using a different USB port or a different computer to test the key.
3. Clear Browser Cache and Cookies
Sometimes, cached data can cause login issues. Clear your browser's cache and cookies, then try logging in again. Here’s how to do it:
In Chrome: Go to Settings > Privacy and security > Clear browsing data.
In Firefox: Go to Options > Privacy & Security > Cookies and Site Data > Clear Data.
4. Try a Different Browser
If the issue persists, try logging in using a different web browser. This can help determine if the problem is browser-specific.
5. Disable Browser Extensions
Some browser extensions can interfere with the login process. Temporarily disable any extensions, especially those related to security or privacy, and try logging in again.
6. Contact GitHub Support
If none of the above solutions work, consider reaching out to GitHub Support. Provide them with details about the error and the steps you've already taken. They may have specific solutions or be able to assist you further.
By following these steps, you should be able to troubleshoot the login issue on your desktop. If you continue to experience problems, GitHub Support will be your best resource for resolving this issue.

**Reply by ebndev** [reply]
Hi @AIBlockchainGuru, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟

**Comment by ebndev** [comment]
Hi @lostinfinite,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
